app/controllers/taxes_controller.rb

```diff
@@ -0,0 +1,79 @@
+class TaxesController < ApplicationController
+  before_action :set_car
+  before_action :set_tax, only: %i[ show edit update destroy ]
+
+  # GET /taxes or /taxes.json
+  def index
+    @taxes = @car.taxes
```

Comment:
if the taxes related to car then it should be under the cars scope, the route should be like `/cars/:car_id/taxes`